src/components/NavigationMailbox.vue

```diff
@@ -401,6 +395,32 @@ export default {
 		subCounter() {
 			return this.subMailboxes.reduce((carry, mb) => carry + mb.unread, 0)
 		},
+		hasRenameAcl() {
+			if (!this.mailbox.myAcls) {
+				return true
+			}
+			const parent = this.$store.getters.getParentMailbox(this.mailbox.databaseId)
+			return this.mailbox.myAcls.indexOf('x') !== -1
+				&& (!parent || parent.myAcls.indexOf('k') !== -1)
```

Comment:
```suggestion
				&& (!parent?.myAcls || parent.myAcls.indexOf('k') !== -1)
```

Probable need to check `myAcls` of parent for nullish values too since it might not have ACLs.